dogfood/main.tf

```diff
@@ -34,6 +45,7 @@ resource "coder_agent" "dev" {
     code-server --auth none --port 13337 &
     sudo service docker start
     coder dotfiles -y 2>&1 | tee  ~/.personalize.log
+    "if [ -n ${var.dotfiles_uri} ]; then coder dotfiles var.dotfiles_uri -y 2>&1 | tee  ~/.personalize.log; fi"
```

Comment:
Unsure if this is the exact way to do this interpolation within the Bash script. Smoke-tested locally using `terraform validate`. cc @bpmct 

Ref: [Conditionals](https://developer.hashicorp.com/terraform/language/v1.1.x/configuration-0-11/interpolation#conditionals)

Reply:
I think you need to quote the variable in order for `-n` to work; I'm not sure if terraform will quote it for you.
At least, shellcheck seems to think so.
Also, I believe double brackets are generally preferred for conditionals over single brackets (ref: https://www.baeldung.com/linux/bash-single-vs-double-brackets)

Reply:
@mafredri Shouldn't we have `set pipefail` in this script?

Reply:
@mtojek Since we're supporting `/bin/sh`, we should avoid `-o pipefail` (it's a bash-ism).